src/shared/api/http-client.ts

```
@@ -0,0 +1,221 @@
+  baseUrl?: string;
+  baseApiParams?: Omit<RequestParams, 'baseUrl' | 'cancelToken' | 'signal'>;
+  securityWorker?: (securityData: SecurityDataType | null) => Promise<RequestParams | void> | RequestParams | void;
+  customFetch?: typeof fetch;
+}
+
+export interface HttpResponse<D extends unknown, E extends unknown = unknown> extends Response {
+  data: D;
+  error: E;
+}
+
+type CancelToken = Symbol | string | number;
```

Biome (1.9.4)

[error] Don't use 'Symbol' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'symbol' instead


(lint/complexity/noBannedTypes)